src/analysis/syntax_doc.ml

```diff
@@ -76,7 +76,7 @@ let get_syntax_doc node : syntax_info =
         {
           name = "Variant Type";
           description =
-            "Represent data that may take on multiple different forms.";
+            "Represents data that may take on multiple different forms.";
```

Comment:
Just a note: this is not plural (it's a verb), this is the third person singlar's `s` 🙂